README.rst

```diff
@@ -111,12 +116,13 @@ You need to use the customized Vagrantfiles provided in the ADB project to set u
 .. _openshift2nulecule: https://github.com/projectatomic/openshift2nulecule/
 .. _Mesos Marathon: https://mesosphere.github.io/marathon/
 
-How do I Install and Run the ADB?
+How do I Install and Run ADB?
 ===========================================================
 
 Below is a quick installation guide using the most common options.
 Detailed `installation instructions`_ are available.
 
+
 1. `Install VirtualBox`_ for your operating system.
```

Comment:
1. Install a virtualization provider.  For many platforms VirtualBox is a good choice.  For Linux, libvirt may be better.